### ISSUE TITLE
Training Details

### ISSUE THREAD

**Issue by CSX777** [liuyuan-pal/LMCNet#6]
Hello!  Thank you so much for releasing the code of this work. It's a nice work.  therefore, i have some interesting questions for you in this work. I am looking for your reply .
Firstly, on the comparison experiment, how many steps do you have  for training the LMC model ?  Is it 500K ?

**Comment by liuyuan-pal** [comment]
Yes, LMCNet is trained 500k steps on the given training dataset.
